rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -241,6 +241,47 @@ void main_window::OnPlayOrPause()
 	}
 }
 
+void main_window::show_boot_error(game_boot_result status)
+{
+	if (status == game_boot_result::no_errors)
+	{
+		return;
+	}
+	QString message;
+	switch (status) {
```

Comment:
I think our formatting puts case in the same tab position as switch.
Also:
```suggestion
	switch (status)
	{
```